CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+- [Cloning a Repo](https://help.github.com/en/desktop/contributing-to-projects/creating-a-pull-request)
+- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+
+***If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).***
+
+## 💥 How to Contribute
+
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
+
+- Take a look at the existing [Issues](https://github.com/eddiejaoude/stargate/issues) or create your own Issues!
+- Wait for the Issue to be assigned to you after which you can start working on it.
+- Fork the Repo and create a branch for any Issue that you are working upon.
+- Create a Pull Request which will be promptly reviewed and suggestions would be added to improve it.
+- Add screenshots to help us know what this script is all about.
+
+
+## ⭐HOW TO MAKE A PULL REQUEST:
+
+**1.** Fork [this](https://github.com/eddiejaoude/stargate) repository.
+Click on the <a href="https://github.com/eddiejaoude/stargate"><img src="https://img.icons8.com/ios/24/000000/code-fork.png"></a> symbol at the top right corner.
+
+**2.** Clone the forked repository.
+
+```bash
+git clone https://github.com/<your-github-username>/stargate
+```
+
+**3.** Navigate to the project directory.
+
+```bash
+cd stargate
+```
+**5.** Create a new branch:
+```bash
+git checkout -b <YourBranchName>
+```
+
+**6.** Make changes in source code.
+
+**7.** Stage your changes and commit
+
+```bash
+git add <file1,file2....>
+```
+```bash
+git commit -m "<your_commit_message>"
+```
+
+**8.** Push your local commits to the remote repo.
+
+```bash
+git push origin <YourBranchName>
+```
+
+**9.** Create a [PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) !
+
+**10.** **Congratulations!** Sit and relax, you've made your contribution to the[Stargate](https://github.com/eddiejaoude/stargate) project.
+
+***:trophy: After this, project leaders and mentors will review the changes and will merge your PR if they add value, or they will suggest any required changes.***
+
+## Style Guides for Git Commit Messages:memo:
+
+### Here's a list of some good to have points, that can add more value to your contribution logs.
+
+- fix: a commit of the type fix patches a bug in your codebase (this correlates with PATCH in semantic versioning).
+- feat: a commit of the type feat introduces a new feature to the codebase (this correlates with MINOR in semantic versioning).
+- BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in semantic        versioning). A BREAKING CHANGE can be part of commits of any type.
```

Comment:
```suggestion
- BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type.
```

Reply:
Still some extra blank spaces left. Please go through all the suggested changes! 🙂